.github/workflows/test-application.yaml

```diff
@@ -27,6 +27,13 @@ jobs:
                           SYMFONY_DEPRECATIONS_HELPER: disabled
 
                     - php-version: '7.4'
+                      lint: true
+                      dependency-versions: 'highest'
+                      tools: 'composer:v2'
+                      env:
+                          SYMFONY_DEPRECATIONS_HELPER: disabled
+
+                    - php-version: '8.0'
```

Comment:
can we add 8.0, 8.1, 8.2 and 8.3 to the matrix also

Reply:
I think 8.3 should be added in a separate MR, there are still some used dependencies that are not 8.3 compatible
https://github.com/sulu/SuluCommentBundle/actions/runs/6268238432/job/17022781860?pr=48